### DISCUSSION TITLE
Benchmarks with meilisearch

### DISCUSSION THREAD

**Discussion by ceddybi** [SeekStorm/SeekStorm#12] [Ideas]
Benchmarks with meilisearch

**Comment by wolfgarbe** [comment]
This would be definitely interesting. The benchmark we are using was developed by Tantivy and Jason Wolfe.
It expects libraries (as opposed to servers) for the test candidates. 
The reasoning is probably to eliminate caching between the different test runs and to exclude the influence of different HTTP stacks to test pure search performance. So, the Meilisearch Rust SDK won't cut it, as it would use the continuously running server in  the background.
For Meilisearch I couldn't find a library crate, only the complete server code. I don't know Meilisearch well enough, to faithfully extract the relevant library code parts from the server code for the benchmark.

That being said, pull requests are welcome at https://github.com/SeekStorm/search-benchmark-game
